entries/c/cpp.edu.json

```diff
@@ -0,0 +1,27 @@
+{
+    "Cal Poly Pomona": {
+        "domain": "cpp.edu",
+        "url": "https://www.cpp.edu",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```

Reply:
For me, when I go to `https://cpp.edu`, it just puts a screen saying "Hmm. We’re having trouble finding that site."

<img width="1680" alt="Screen Shot 2022-11-10 at 11 00 14" src="https://user-images.githubusercontent.com/16967687/201183118-475f134e-e006-4fb1-9e6d-58a42a3c4fc2.png">

Reply:
Ah you're right the HTTPS version of cpp.edu doesn't work. My bad

Reply:
No worries